src/pymsm/multi_state_competing_risks_model.py

```diff
@@ -340,7 +340,7 @@ def run_monte_carlo_simulation(
 
     def _one_monte_carlo_run(
         self,
-        sample_covariates,
+        sample_covariates: np.ndarray,
```

Comment:
Can we also allow pd.Series (and then call .values if so)?. How do you type hint such a thing?

Reply:
To type two different option one can use Union[np.ndarray, pd.Series] - and then check in the code the type and treat the different option (either use .values or not)